recipes/nfir/all/conandata.yml

```
@@ -0,0 +1,11 @@
+sources:
+  "0.2.0":
+    url: "https://codeload.github.com/tomas-krupa/NFIR/zip/refs/tags/0.2.0"
+    sha256: "836ad87ae6a25c9a7f361a854d30c25560e7af833a5daeb906e4867f93b16276"
```

```suggestion
    url: "https://github.com/usnistgov/NFIR/archive/b670b673f8c397f37b231d16bc8b41f934d05841.tar.gz"
    sha256: "0bce5cd844560f82968540b0ae27248973bc1f3bfccadfc80e687f129a5588e8"
```
You can use the latest commit from the original repo and set the version to `0.2.0.cci.20241122` to account for no exact releases being made.